data-prepper-plugins/parse-json-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/parse/ion/ParseIonProcessorConfig.java

```diff
@@ -38,6 +41,14 @@ public class ParseIonProcessorConfig implements CommonParseConfig {
     @JsonProperty
     private boolean deleteSource = false;
 
+    @JsonProperty("handle_failed_events")
+    @JsonPropertyDescription("Determines how to handle events with ION processing errors. Options include 'skip', " +
+            "which will log the error and send the Event downstream to the next processor, and 'skip_silently', " +
+            "which will send the Event downstream to the next processor without logging the error. " +
+            "Default is 'skip'.")
+    @NotNull
+    private HandleFailedEventsOption handleFailedEventsOption = HandleFailedEventsOption.SKIP;
```

Comment:
This means, default behavior is not changed, right? ie, the failures are logged and skipped. This also, means customers will see this issue, unless they explicitly change this. Is this what is agreed upon? Should we set it to SKIP_SILENTLY in OSI?

Reply:
Yes that's right default is to log and skip still